tests/system/action/projector/test_project_preview.py

```diff
@@ -0,0 +1,83 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class ProjectorProjectPreview(BaseActionTestCase):
+    def setUp(self) -> None:
+        super().setUp()
+        self.set_models(
+            {
+                "meeting/1": {},
+                "projector/2": {"meeting_id": 1},
+                "projector/3": {
+                    "current_projection_ids": [1, 2],
+                    "preview_projection_ids": [3, 4],
+                    "history_projection_ids": [6],
+                    "meeting_id": 1,
+                },
+                "projector/4": {
+                    "current_projection_ids": [],
+                    "preview_projection_ids": [5],
+                    "history_projection_ids": [],
+                    "meeting_id": 1,
+                },
+                "projection/1": {
+                    "current_projector_id": 3,
+                    "meeting_id": 1,
+                    "weight": 100,
+                    "stable": True,
+                },
+                "projection/2": {
+                    "current_projector_id": 3,
+                    "meeting_id": 1,
+                    "weight": 98,
+                },
+                "projection/3": {
+                    "preview_projector_id": 3,
+                    "meeting_id": 1,
+                    "weight": 99,
+                },
+                "projection/4": {
+                    "preview_projector_id": 3,
+                    "meeting_id": 1,
+                    "weight": 100,
+                },
+                "projection/5": {
```

Comment:
Please switch the order, so the projections are ordered by the projector. This makes it easier for a reader of the tests.

Reply:
Order of the projections changed. So it is easier to read.